OpenRVS/classes/OpenClientBeaconReceiver.uc

```diff
@@ -40,12 +40,12 @@ event ReceivedText(IpAddr Addr,string Text)
 		{
 			//if we got to this stage, it's a REPORT response
 			//start szthirdword at the first symbol for GrabOption() to work
-			szThirdWord = mid(szThirdWord,InStr(szThirdWord,"�"));
+			szThirdWord = mid(szThirdWord,InStr(szThirdWord,"¶"));
 			//debug:
-//			log(left(szThirdWord,20));
+			class'OpenLogger'.static.DebugLog(left(szThirdWord,20));

```

Comment:
In this function we should add a descriptive string to the debug log - otherwise if we enable debug logging, we'll get the output of this parser inserted in the log without any sort of context as to what they are.  EG:
class'OpenLogger'.static.DebugLog("*** debug: szThirdWord first 20 characters: " $ left(szThirdWord,20));
Adds "*** debug: szThirdWord left 20: " to the line in the log file.